library/src/main/java/com/schibsted/spain/barista/internal/matcher/BackgroundMatcher.kt

```diff
@@ -0,0 +1,65 @@
+package com.schibsted.spain.barista.internal.matcher
+
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.drawable.Drawable
+import android.support.annotation.DrawableRes
+import android.view.View
+import android.widget.ImageView
+import com.schibsted.spain.barista.internal.util.BitmapComparator
+import com.schibsted.spain.barista.internal.util.DrawableToBitmapConverter
+import org.hamcrest.Description
+import org.hamcrest.TypeSafeMatcher
+
+class BackgroundMatcher private constructor(@DrawableRes private val expectedDrawableRes: Int) : TypeSafeMatcher<View>(View::class.java) {
```

Comment:
Don't this class and the Drawable matcher are basically the same? 
We should try to DRY and extract the common code to some base matcher that we can reuse from both, and avoid duplicating code.

Reply:
Yes, but they have some differences. Can we do that work in another PR? It's not as simple as it would seem with a quick look to the code :·/